modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -235,8 +235,6 @@ export default {
     await this.getPieces();
     await this.getAllPiecesTotal();
 
-    this.batchOperations = this.flattenOperations();
```

Comment:
This is still used in `handleBatchAction`. The purpose was to support finding the operation details from the operation name easily. We could check multiple levels of operations in `handleBatchAction` instead (top level then group operations), but we can't simply remove this.

Reply:
Don't you think it would be easier to directly pass needed informations to `handleBatchAction` since we have everything available when confirming the modal instead of recomputing? It would avoid to compute the same thing multiple times. Maybe it would be better to merge the modal one first to integrate with.